packages/utils/telemetry-utils/src/error.ts

```diff
@@ -47,9 +54,138 @@ export class UsageError extends LoggingError implements IUsageError, IFluidError
  */
 export class DataCorruptionError extends LoggingError implements IErrorBase, IFluidErrorBase {
 	readonly errorType = FluidErrorTypes.dataCorruptionError;
-	readonly canRetry = false;
+	// readonly canRetry = false;
```

Comment:
why this change?

Reply:
I hadn't intended to push that. Good catch - thanks!

Reply:
```suggestion
	readonly canRetry = false;
```